index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+require_once 'functions.php';
+require_once 'dbConnect.php';
+
+$db = connectdb();
+
+
+$shinkansens = fetchData($db);
+
+$display = '';
+foreach($shinkansens as $shinkansen) {
+    $display .= collectionBox($shinkansen);
```

Comment:
collection box name not the most semantic - maybe something like displayTrains()